scripts/simple_raco_execution.py

```diff
@@ -0,0 +1,87 @@
+
+'''
+A notional raco-only script for executing a MyriaL query
```

Comment:
Is this meant to be executable? I think we should make it so.

Needs `raco.convenience`